src/renderer/shims/LocalShim.ts

```diff
@@ -81,12 +81,23 @@ export class LocalShim implements SimulatorInterface {
     public Aircraft = {}
 
     public GetStoredData = (key: string): any => {
-        if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
-            return 2;
+        try {
+            return window.localStorage.getItem(key);
+        } catch (error) {
+            console.log(error);
         }
-        return 0;
+        return '';
     }
 
-    public SetStoredData = () => {
+    public SetStoredData = (key: string, value: any): any => {
+        if (typeof value !== 'string') {
+            throw Error('SetStoredData: Illegal type of value! value must be of type string');
```

Comment:
How come you check to see if the type of the argument made for value is not of type string? Why don't you just change the type of the value parameter?

Reply:
Because that would not match what the sim does => ACE would not be as useful for debugging.

Reply:
> Because that would not match what the sim does => ACE would not be as useful for debugging.

Thanks for explaining, I just checked the ts typing that is currently in place for the function and it checks out 👍 : 
function SetStoredData(key: string, data: any): any;

Reply:
Yep, the sim actually silently fails to set the value if you pass anything that's not a string (https://github.com/flybywiresim/ace/pull/12#discussion_r762796413).